### DISCUSSION TITLE
Sorting with vanilla datatable

### DISCUSSION THREAD

**Discussion by daniel-nedelcu** [carbon-design-system/carbon#10623] [Help] [answered]
does anybody has a working example of datatable sorting using vanilla js ?

**Comment by wtrudell360** [comment]
Are you not able to get the default sorting to work or do you need custom sorting?

https://react.carbondesignsystem.com/?path=/docs/components-datatable-sorting--usage#custom-sorting  programmatic and custom sorting.

**Reply by daniel-nedelcu** [reply]
I'm not looking at the react components, I need to get the sorting done with vanilla js https://the-carbon-components.netlify.app/?nav=data-table

**Comment by tay1orjones** [comment, marked as answer]
Hi there! :wave: Thanks for reaching out about this issue with the Vanilla JS set of components. Unfortunately, the Vanilla JS components have been put in maintenance mode since our v10 release as our team has shifted focus to React. We try and help out as best we can but unfortunately no new development is occurring in that part of the codebase.

For teams looking to not use a framework for components, we recommend switching over to the Web Components library as that will be the strategy going forward in this area.
